docs/versioned_docs/v0.8/basics/components.md

```diff
@@ -115,3 +118,56 @@ fn MyComponent(cx: Scope) -> View<G> {
     // ...
 }
 ```
+
+### Nested components
```

Comment:
```suggestion
### Component children
```
This represents better what this does. "Nested components" might be confused with the children of the component.

Reply:
Changed.